test/passes/O1_print-stack-ir.txt

```diff
@@ -1,5 +1,30 @@
-$stacky-help:
- (no stack ir)
+(module
```

Comment:
So this prints the module twice? Should it?
I'd expect we should just print once, with valid wat if we have stack IR, and folded form if we don't. Binaryen's wast parser can handle flat wat, right?

Reply:
I think it's the test runner that is confusing here: when we run tests in `/passes/`, we run the requested passes plus `--print`, so we always print at the end (so that the majority of tests don't need to specify `--print`). For this tests we also manually print using `--print-stack-ir`, so the module it appears twice.